imports/ui/ranking/Ranking.html

```diff
@@ -7,7 +7,7 @@
             <th class="elo">ELO</th>
             <th class="games-played">Games Played</th>
         </tr>
-        {{ #each player in (getPlayers sortBy) }}
+        {{ #each player in (getPlayers sortBy format) }}
```

Comment:
I know it's a bit late, but maybe comments would make maintaining all this crap easier?